ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -24,7 +24,7 @@ base_images:
     namespace: ci
     tag: "4.15"
   tools:
-    name: "4.15"
+    name: "4.16"
```

Comment:
@liangxia shall we update the tool version to target version? because I saw the existing upgrade ci job is using the source cluster version, e.g: `openshift-openshift-tests-private-release-4.15__arm64-nightly-4.15-upgrade-from-stable-4.14.yaml`

Reply:
I think those changes are unexpected.